src/policy/rbf.cpp

```diff
@@ -7,7 +7,7 @@
 bool SignalsOptInRBF(const CTransaction &tx)
 {
     for (const CTxIn &txin : tx.vin) {
-        if (txin.nSequence < std::numeric_limits<unsigned int>::max()-1) {
+        if (txin.nSequence < CTxIn::SEQUENCE_FINAL - 1) {
```

Comment:
could `if (txin.nSequence <= MAX_BIP125_RBF_SEQUENCE) {` ?

Reply:
This was suggested by gmexwell [here](https://github.com/bitcoin/bitcoin/pull/14463#discussion_r224850088)